apps/wallet/src/ui/app/components/active-coins-card/CoinItem.tsx

```diff
@@ -29,19 +29,23 @@ export function CoinItem({
 		<div className="flex gap-2.5 w-full py-3 pl-1.5 pr-2 justify-center items-center rounded hover:bg-sui/10">
 			<CoinIcon coinType={coinType} size={isActive ? 'sm' : 'md'} />
 			<div className="flex flex-1 gap-1.5 justify-between items-center">
-				<div className="flex flex-col gap-1.5">
+				<div className="block max-w-token-width">
 					<Text variant="body" color="gray-90" weight="semibold" truncate>
 						{coinMeta?.name || symbol} {isActive ? 'available' : ''}
 					</Text>
 					{!isActive && !subtitle ? (
-						<Text variant="subtitle" color="steel-dark" weight="medium">
-							{symbol}
-						</Text>
+						<div className="mt-1.5">
```

Comment:
I would probably have tried to keep flex to avoid setting margin here

Reply:
So the reason I have to do `block` is because `truncate` class does not work when the parent is not given a fixed size + block. Refer to this post here https://stackoverflow.com/questions/71093772/how-to-truncate-text-in-tailwindcss

Currently our texts will overflow otherwise

CC: @pchrysochoidis

Reply:
That's fine, just wanted to avoid margins if possible